src/components/d2l-help/HowToEditBannerImage.js

```diff
@@ -0,0 +1,41 @@
+import styles from "../../styles/d2l.module.css";
+
+export default function HowToEditBannerImage() {
+  return (
+    <>
+      <section id="toc-how-to-edit-banner-image">
+        <h2 id="how-to-edit-banner-image" className="section-top anchor">How to Edit the Banner Image</h2>
+        <p className="mb-8">Learn how to edit the banner image that is located at the top of the web page.</p>
+				<ol className={styles.d2lInstructions}>
+					<li><p>Click the small caret icon next to the page title.</p>
+						<img src="/images/d2l-help/edit-banner-image/step1.png"></img>
+					</li>
+					<li><p>From the dropdown, select the "Edit HTML" option.</p>
+						<img src="/images/d2l-help/edit-banner-image/step2.png"></img>
+					</li>
+					<li><p>The web page will switch to edit mode. Click on the banner image area to see a blue outline indicating that it's selected.</p>
+						<img src="/images/d2l-help/edit-banner-image/step3.png"></img>
+					</li>
+					<li><p>When the banner image area is selected, right-click to see a box labeled "Image..." and click it.</p>
+						<img src="/images/d2l-help/edit-banner-image/step4.png"></img>
+					</li>
+					<li><p>A dialogue box will appear. Edit the source to change the image.</p>
```

Comment:
The image will be imported from Course Files and not a URL. I am not sure that we can import an image from Course Files via the visual editor.

Reply:
I see. I am not sure what the workflow is like for IDs. I feel like sometimes with this documentation, we should run it past an ID because I'm not sure if this is their actual work flow. 

I'll reach out to an ID to see how they typically would add a banner image here and restructure the docs.